src/Sylius/Behat/Context/Ui/Shop/Checkout/CheckoutShippingContext.php

```diff
@@ -31,11 +31,16 @@ public function __construct(
     }
 
     /**
+     * @Given the visitor has proceeded :shippingMethodName shipping method
+     * @Given the customer has proceeded :shippingMethodName shipping method
+     * @Given the visitor proceed with :shippingMethodName shipping method
+     * @Given the customer proceed with :shippingMethodName shipping method
      * @Given I completed the shipping step with :shippingMethodName shipping method
+     * @Given I have proceeded with :shippingMethodName shipping method
      * @Given I have proceeded selecting :shippingMethodName shipping method
      * @When I proceed with :shippingMethodName shipping method
      */
-    public function iHaveProceededSelectingShippingMethod(string $shippingMethodName): void
+    public function iHaveProceededWithSelectingShippingMethod(string $shippingMethodName): void
```

Comment:
If I see correctly, this method is used in other places, so you need to adjust its name there